webui/react/src/components/Navigation.tsx

```diff
@@ -34,6 +40,13 @@ const Navigation: React.FC<Props> = ({ children }) => {
   usePolling(fetchPinnedWorkspaces);
   usePolling(fetchUserSettings, { interval: 60000 });
 
+  useEffect(() => {
```

Comment:
This was previously happening inside a store action. This actually caused the effect to double up on one page. Makes more sense for it to live here.